changelog.d/610.misc

```diff
@@ -0,0 +1 @@
+Docker images are now automatically build and published via GitHub Actions, replacing DockerHub Autobuilds. 
```

Comment:
Do we need to remove any file to stop DockerHub Autobuilds?

Reply:
No files, but I've configured dockerhub to stop doing builds.